qa/sc_cert_fee_conf.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env python2
+import time
+from decimal import Decimal
+
+from SidechainTestFramework.sc_boostrap_info import SCNodeConfiguration, SCCreationInfo, MCConnectionInfo, \
+    SCNetworkConfiguration
+from SidechainTestFramework.sc_test_framework import SidechainTestFramework
+from test_framework.util import assert_false, assert_true, start_nodes, \
+    websocket_port_by_mc_node_index
+from SidechainTestFramework.scutil import bootstrap_sidechain_nodes, \
+    start_sc_nodes, connect_sc_nodes, generate_next_blocks, generate_next_block
+from SidechainTestFramework.sc_forging_util import *
+
+"""
+Check the bootstrap feature.
+
+Configuration: bootstrap 1 SC node and start it with genesis info extracted from a mainchain node.
+    - Mine some blocks to reach hard fork
+    - Create 2 SC node(first nod with constant fee, second nod with automatic fee computation)
+
+Test:
+    - generate MC and SC blocks to reach the end of the Withdrawal epoch 0
+    - generate one more MC and SC block accordingly and await for certificate submission to MC node mempool
```

Comment:
Specify that cert is generated by the first node with fixed fee.
Similar below, that we disable node 1 and enable node 2 cert submission. When we generate epoch 1 cert - that we use sc node 2 with automatic fee.